packages/patapata_core/lib/src/analytics.dart

```diff
@@ -486,14 +486,11 @@ class AnalyticsContextProvider extends SingleChildStatelessWidget {
   /// Provide an [analyticsContext] and pass the widget to be wrapped as [child].
   /// Optionally, use the [reset] flag to specify whether to reset the [analyticsContext].
   const AnalyticsContextProvider({
-    Key? key,
+    super.key,
     required this.analyticsContext,
-    required Widget child,
+    required Widget super.child,
```

Comment:
This comment is for the entire PR. But when you use `super` you don't need to declare the type, just like the `key` above.